test/utils/client.js

```diff
@@ -42,23 +42,23 @@ async function createConsumer(registry, identity, accountantId) {
 }
 
 function createProvider(identity, accountant) {
-    const state = { 
+    const state = {
         invoices: {
-        // "invoiceId": {
-        //     agreementID: 1,
-        //     agreementTotal: 0,
-        //     r: 'abc',
-        //     // paid: false,
-        //     exchangeMessage: {}
-        // }
-        }, 
+            // "invoiceId": {
```

Comment:
remove dead code?

Reply:
This is comment of how object structure should look like. Unfortunately JavaScript is not typed and I'm lazy to migrate into TypeScript.